src/server.cpp

```diff
@@ -72,56 +72,197 @@ int init() {
 	createServerPipe(serverPath);
 	return 0;
 }
+/*
+ * @brief checks sign before string/integer
+ *
+ * returns:
+ * 		0: no sign (==)
+ * 		1: >
+ * 		2: >=
+ * 		3: <
+ * 		4: <=
+ * 		5: *
+ */
+int checkIfSign(string pattern) {
+	if (pattern.size() > 1) {
+		if (pattern[0] == '>' && !(pattern[1] == '='))
+			return 1;
+		else if (pattern[0] == '>' && pattern[1] == '=')
+			return 2;
+		else if (pattern[0] == '<' && !(pattern[1] == '='))
+			return 3;
+		else if (pattern[0] == '<' && pattern[1] == '=')
+			return 4;
+	}
+	if (pattern.size() == 1 && pattern[0] == '*')
+		return 5;
+	return 0; //only 1 char, cannot be sign
+}
+
+/*
+ * @brief compares string lexically
+ *
+ * returns:
+ * 		-1: str1 < str2
+ * 		 0: str1 == str2
+ * 		 1: str1 > str2
+ */
+int compareLex(string str1, string str2) {
```

Comment:
This function (equivalent) is provided by std::string already by simple using >, < and == operators. There is no need to compare each character, not even to write it at all

Reply:
it's because of '*' char

Reply:
ok, you're right